README.rst

```diff
@@ -1,110 +1,92 @@
-urllib3
-=======
+hip
+===
 
-.. image:: https://travis-ci.org/urllib3/urllib3.svg?branch=master
+.. image:: https://travis-ci.org/python-trio/hip.svg?branch=master
         :alt: Build status on Travis
-        :target: https://travis-ci.org/urllib3/urllib3
+        :target: https://travis-ci.org/python-trio/hip
 
-.. image:: https://img.shields.io/appveyor/ci/urllib3/urllib3/master.svg
-        :alt: Build status on AppVeyor
-        :target: https://ci.appveyor.com/project/urllib3/urllib3
+.. image:: https://github.com/python-trio/hip/workflows/CI/badge.svg
+        :alt: Build status on GitHub Actions
+        :target: https://github.com/python-trio/hip/actions
 
-.. image:: https://readthedocs.org/projects/urllib3/badge/?version=latest
-        :alt: Documentation Status
-        :target: https://urllib3.readthedocs.io/en/latest/
-
-.. image:: https://img.shields.io/codecov/c/github/urllib3/urllib3.svg
+.. image:: https://img.shields.io/codecov/c/github/python-trio/hip.svg
         :alt: Coverage Status
-        :target: https://codecov.io/gh/urllib3/urllib3
+        :target: https://codecov.io/gh/python-trio/hip
```

Comment:
Here the img shield badge is showing 100% coverage but codecov is showing 99.6%, maybe we should use the badge by codecov

Reply:
Oh that's bad, but do you know where we can find a badge with the correct information?

Reply:
For Trio we're using this URL: https://codecov.io/gh/python-trio/trio/branch/master/graph/badge.svg

So maybe: https://codecov.io/gh/python-trio/hip/branch/master/graph/badge.svg
Which looks like: ![coverage](https://codecov.io/gh/python-trio/hip/branch/master/graph/badge.svg)

Reply:
Thanks! Fixed.